### PR TITLE
TBS Adds a recovery email topic to custom email guide

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description
- **What's changed?** Adds an already-published topic from the h.o.c. to the d.o.c. in the "Customize email notifications" guide.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-490773](https://oktainc.atlassian.net/browse/OKTA-490773)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 1]
```diff
@@ -253,6 +253,38 @@ Variables used for function parameters must match the function data type. For ex
 | escapeHtml(String html)                                       | Escapes the characters in the provided string using HTML entities                                                             |
 | escapeHtmlAttr(String html)                                   | Encodes data for use in HTML attributes                                                                           |
 
+## Customize a forgotten password recovery email
+
+Okta can send an email notification to an end user who has forgotten their password. They must follow the provided link to complete the forgot password process.
+
+If you are using an [embedded authentication](/docs/concepts/redirect-vs-embedded/) deployment model for your application's sign-in flow, you can create a customized email magic link for your **Forgot Password** email template.
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:260]: why is embedded model important here? is deployment model relevant in terms of customizing this email template?

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 2]
```diff
@@ -253,6 +253,38 @@ Variables used for function parameters must match the function data type. For ex
...
+
+The option to send a forgotten password recovery email with a custom link is available only to OIDC applications on Identity Engine organizations.
+
+You can customize any default email template. However, when you edit a default template, Okta stops sending the default message in other languages. You are responsible for adding translations of your customized message.
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:264]: can we move this paragraph to a top level section if it isn't already apparent? this applies to all email templates, not just this forgot pw template

**Review by albertchen-okta — CHANGES_REQUESTED** [review]
Overall I'm not a huge fan of this section period, most of these instructions apply to all email templates, and the variables that are compatible with each template are already called out in https://developer.okta.com/docs/guides/custom-email/main/#use-vtl-variables so I'm not sure why this email template deserves a special call out

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:260]: @albertchen-okta According to @zeusandrade-okta :
> admins can customize the templates as desired
> if they want to have a custom magic link in the template, it should be for embedded

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md (hunk 2 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md]: Done :)

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md (hunk 1 above)
Comment (albertchen-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:262]: is he saying redirect customers CANNOT use custom magic link? makes no sense but im not close to the feature

**Pull request closed without merging**